masonry_winit/examples/custom_widget.rs

```diff
@@ -156,8 +158,7 @@ impl Widget for CustomWidget {
         // Let's burn some CPU to make a (partially transparent) image buffer
         let image_data = make_image_data(256, 256);
         let image_data = Image::new(image_data.into(), ImageFormat::Rgba8, 256, 256);
-        let transform = ObjectFit::Fill.affine_to_fill(ctx.size(), size);
-        scene.draw_image(&image_data, transform);
+        scene.draw_image(&image_data, Affine::IDENTITY);
```

Comment:
It's not clear why this change was necessary, since you have a fix in `affine_to_fill`

Reply:
Hmm, I'd probably expect something like `ObjectFit::affine_to_fill(ObjectFit::Fill, ctx.size(), Size::new(256., 256.))`, maybe?